src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -177,8 +243,15 @@ private static FermiLiteAssembly interpretAssemblyData( final ByteBuffer assembl
         return new FermiLiteAssembly(contigs);
     }
 
-    // these should be called in succession by the same thread
-    private static native ByteBuffer createAssemblyData( final ByteBuffer readData );
-    private static native void destroyAssemblyData( final ByteBuffer assemblyData );
+    private ByteBuffer getOpts() {
```

Comment:
👍 I thought about asking you to do that, but I figured I'd nitpicked enough already.